rest/route/project.go

```diff
@@ -1120,5 +1122,10 @@ func (h *projectVarsPutHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(err,
 			"error updating projects with matching keys"))
 	}
+	_, err = artifact.RotateSecrets(h.replaceVars.ToReplace, h.replaceVars.Replacement, h.replaceVars.RotateFiles)
```

Comment:
> I think this isn't right; we just shouldn't call RotateSecrets at all if RotateFiles isn't true, but RotateSecrets should still consider DryRun.

I'm not sure why this comment went away 🤔   this still needs to be addressed; rotateFiles shouldn't be treated as dry run.

Reply:
huh i dont know why i didnt see but i changed it so that it would still dryrun but only go into it if rotatefiles was set